x/interchainstaking/handler.go

```diff
@@ -10,28 +10,6 @@ import (
 	"github.com/ingenuity-build/quicksilver/x/interchainstaking/types"
 )
 
-// NewHandler returns a handler for interchainstaking module messages
```

Comment:
I thought this was still used for amino messages? Or is it deprecated entirely now?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/9650

Added in `v0.46.0`

Reply:
thank you :)